### ISSUE TITLE
pip looking for wrong python3-ldap version

### ISSUE THREAD

**Issue by PrivateStaticVoid** [etianen/django-python3-ldap#5]
Hi,

I'm having an issue installing through pip. It's looking for python3-ldap version 0.9.5.3, which doesn't seem to exist (`pip install python3-ldap==0.9.5.3` fails for me as well). Is there any way to use a different python3-ldap version?

```
Downloading/unpacking python3-ldap==0.9.5.3 (from django-python3-ldap)
  Could not find a version that satisfies the requirement python3-ldap==0.9.5.3 (from django-python3-ldap) (from versions: 0.9.3.5, 0.9.4.2, 0.9.5.4, 0.9.6.2, 0.9.7, 0.8.3, 0.9.1, 0.9.2.2, 0.9.3.5, 0.
9.4.2, 0.9.5.4, 0.9.6.2, 0.9.7)
Cleaning up...
```

Thanks!

**Comment by PrivateStaticVoid** [comment]
Installs successfully using pip3 instead of pip.

**Comment by etianen** [comment]
This is fixed in #2fe54797df8a7efbcb58e14de72b415069af7eb4
